packages/env/lib/create-docker-compose-config.js

```diff
@@ -0,0 +1,55 @@
+module.exports = function createDockerComposeConfig(
+	pluginPath,
+	pluginName,
+	pluginTestsPath
+) {
+	const commonVolumes = `
+      - ${ pluginPath }/:/var/www/html/wp-content/plugins/${ pluginName }/
+      - ${ pluginPath }${ pluginTestsPath }/e2e-tests/mu-plugins/:/var/www/html/wp-content/mu-plugins/
+      - ${ pluginPath }${ pluginTestsPath }/e2e-tests/plugins/:/var/www/html/wp-content/plugins/${ pluginName }-test-plugins/`;
+	const volumes = `
+      - ${ pluginPath }/../${ pluginName }-wordpress/:/var/www/html/${ commonVolumes }`;
+	const testsVolumes = `
+      - tests-wordpress:/var/www/html/${ commonVolumes }`;
+	return `version: '2'
+volumes:
+  tests-wordpress:
+services:
+  mysql:
+    environment:
+      MYSQL_ROOT_PASSWORD: password
+    image: mariadb
+  wordpress:
+    depends_on:
+      - mysql
+    environment:
+      WORDPRESS_DEBUG: 1cq
```

Comment:
What is this value?

Reply:
I copied it from another setup. Riad's I think.

Reply:
I don't see any docs for it.

Should we have `WP_DEBUG=true` instead?

Reply:
It looks like a vim typo 😛

Reply:
Yeah, we need to change it. `WP_DEBUG=true` is correct right?

Reply:
Based on the documentation for the environment variable, it would be `1`, which seems consistent at least with part of what was written here.

https://hub.docker.com/_/wordpress

The actual constant it controls would have a value of `true` or `false`:

https://wordpress.org/support/article/debugging-in-wordpress/

I guess it's also a question of whether it's intended to enable debugging mode. I would guess based on its initial presence here, and the fact this is intended for development, it would be safe to say "yes".

Reply:
>-e WORDPRESS_DEBUG=1 (defaults to disabled, non-empty value will enable WP_DEBUG in wp-config.php)

It's already working. It just needs a non-empty value.

Still opened a PR to be tidy: #19218.